changelog/@unreleased/pr-3119.v2.yml

```
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Make baseline configuration cacheable
```

```suggestion
  description: Make `gradle-baseline` configuration-cache compatible
```

This phrasing is somewhat more accurate and is similar to the nomenclature other plugins appear to be using.